sdk/loadtestservice/Azure.Developer.LoadTesting/tests/Samples/Sample1_HelloWorld.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
let's remove the snippet region until we have a real sample.

Reply:
Removed samples folder from tests, I hope it is fine?